lib/internal/Magento/Framework/ObjectManager/Config/Config.php

```diff
@@ -335,19 +343,4 @@ public function getPreferences()
     {
         return $this->_preferences;
     }
-
-    /**
-     * Get serializer
-     *
-     * @return \Magento\Framework\Serialize\SerializerInterface
-     * @deprecated 101.0.0
-     */
-    private function getSerializer()
```

Comment:
Is there any other reason to move it to constructor except static test failure?
If not - please rollback this change

Reply:
Hi @xmav ! I think some changes reverted before and only this remains. I reverted changes. 
Thanks for your CR.